relationships.md

```diff
@@ -0,0 +1,280 @@
+# Relationship Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint contains the actual relationships connecting 2 items.
+It uses a [relationship type](relationshiptypes.md) and the 2 items with HAL links
+
+## Main Endpoint
+**/api/core/relationships**
+
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes
+
+## Single Relationship
+**/api/core/relationships/<:id>**
+
+A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationships/530
+The leftId and rightId parameter in JSON may still be present, but they will be removed soon (requires other PRs to be merged first)
+
+```json
+{
+  "id": 530,
+  "relationshipTypeId": 0,
+  "leftPlace": 1,
+  "rightPlace": 1,
+  "type": "relationship",
+  "_links": {
+    "relationshipType": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+    },
+    "self": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/relationships/530"
+    },
+    "leftItem": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/items/e98b0f27-5c19-49a0-960d-eb6ad5287067"
+    },
+    "rightItem": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/items/0ffbee3f-e7ea-42bc-92fe-2fbef1a52c0f"
+    }
+  },
+  "_embedded": {
+    "relationshipType": {
+      "id": 1,
+      "leftLabel": "isAuthorOfPublication",
+      "rightLabel": "isPublicationOfAuthor",
+      "leftMinCardinality": 0,
+      "leftMaxCardinality": 2147483647,
```

Comment:
This should be null, or simply left out

Reply:
The cardinality values have been updated